### PR TITLE
[BPK-4381] Migrate checkbox and chips pages to mdx

### PR DESCRIPTION
**Pull request by shaundon**

### CODE REVIEW

File: docs/src/pages/components/Checkbox/WebCheckbox.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  <IndeterminateExample />
+</PresentationBlock>
+
+## Disabled
+
+<PresentationBlock>
+  <DisabledExample />
+</PresentationBlock>
+
+## Disabled
```
Comment (georgegillams) [inline docs/src/pages/components/Checkbox/WebCheckbox.mdx:43]:
```suggestion
## Invalid
```

File: docs/src/pages/components/Checkbox/WebCheckbox.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+<PresentationBlock>
+  <InvalidExample />
+</PresentationBlock>
+
+## White
+
+You can change the label to white for displaying on dark backgrounds.
+
+<PresentationBlock>
```
Comment (georgegillams) [inline docs/src/pages/components/Checkbox/WebCheckbox.mdx:53]: This should have a dark background

File: docs/src/pages/components/Checkbox/WebCheckbox.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  <WhiteExample />
+</PresentationBlock>
+
+## Small label
+
+Use the `smallLabel` prop to reduce the font size of the label.
+
+<PresentationBlock>
+  <SmallLabelExample />
+</PresentationBlock>
```
Comment (georgegillams) [inline docs/src/pages/components/Checkbox/WebCheckbox.mdx:63]: Maybe we shouldn't document this as it's not too accessible.
Why is it even a thing?

**Review by georgegillams — CHANGES_REQUESTED** [review]
Unrelated to this PR, but disabled chips should have a different colour for pressed. [See here](https://backpack.github.io/prs/904/components/chip/?platform=web#selectable)

File: docs/src/pages/components/Checkbox/WebCheckbox.mdx (hunk 3 above)
Comment (shaundon) [reply docs/src/pages/components/Checkbox/WebCheckbox.mdx:63]: No idea, I think if it exists it should be documented though, then we remove it later?

Comment (shaundon) [reply docs/src/pages/components/Checkbox/WebCheckbox.mdx:63]: Changed my mind, I'll remove it from the docs

**Review by georgegillams — APPROVED** [review]

**Pull request merged**